### DISCUSSION TITLE
Cross seed on Unraid HELP!!!

### DISCUSSION THREAD

**Discussion by kamikaze4441** [cross-seed/cross-seed#797] [Q&A]
can someone help me with cross seed v6 on unraid. At the moment i think there's something wrong with torrent dir (i would like to use Data-Based Matching) and cross seed isn't injecting the stuff i have in my torrent downloads folder.

is someone able share their config and docker template screenshots

here is config https://imgur.com/a/5y6dHrU

**Comment by zakkarry** [comment]
I highly recommend you read the documentation at www.cross-seed.org - there is a tutorials section, inside it has an unraid section I put screenshots and notes on, but for v6 use that in tandem with the migration guide.

In the top right there is a Discord button, it would be easiest to help you if you popped on if that's possible.

Otherwise, the templates are there.

I would say if you're intending on cross-seeding data from Radarr/Sonarr, you'd want to mount your dataDir as /data to /data - and then it should allow you to add `dataDirs: ['/data/movies']` to your config and search your radarr library (for instance)

IF you have specific questions, feel free to ask here, but as I said it would be easiest to handle this in a live conversation.

**Comment by kamikaze4441** [comment]
thanks for the response!! I'll reach out on discord tomorrow, gotta sleep now  
（づ￣3￣）づ╭❤️～
